src/main/java/io/securecodebox/persistence/defectdojo/service/GenericDefectDojoService.java

```diff
@@ -73,14 +75,19 @@ public final T get(long id) {
 
     final var url = this.config.getUrl() + API_PREFIX + this.getUrlPath() + "/" + id;
     log.debug("Requesting URL: {}", url);
-    ResponseEntity<T> response = restTemplate.exchange(
-      url,
-      HttpMethod.GET,
-      payload,
-      getModelClass()
-    );
-
-    return response.getBody();
+    try {
+      ResponseEntity<T> response = restTemplate.exchange(
```

Comment:
Did you intentionally only add this for the GET requests?
The error handling for the other http methods seems to be the same

Reply:
🤔No. This means that any error handling is missing on non-GET bc I simply grepped `HttpClientErrorException` 😬